packages/trpc/server/routers/viewer/webhook/getByViewer.handler.ts

```diff
@@ -80,7 +89,8 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
     throw new TRPCError({ code: "INTERNAL_SERVER_ERROR" });
   }
 
-  const userWebhooks = user.webhooks;
+  let userWebhooks = user.webhooks;
+  userWebhooks = userWebhooks.filter(filterWebhooksWithInvalidSubscriptionUrl);
```

Comment:
Webhooks have an app field. We can just filter out all webhooks that have zapier in the app field

Reply:
@CarinaWolli  Thank you for your quick review. I've made the suggested changes. Please review the updated code.